core-bundle/src/Resources/contao/classes/DataContainer.php

```diff
@@ -1726,7 +1737,7 @@ public function generateRecordLabel(array $row, string $table = null, bool $prot
 			}
 			elseif (($GLOBALS['TL_DCA'][$table]['fields'][$v]['eval']['isBoolean'] ?? null) || (($GLOBALS['TL_DCA'][$table]['fields'][$v]['inputType'] ?? null) == 'checkbox' && !($GLOBALS['TL_DCA'][$table]['fields'][$v]['eval']['multiple'] ?? null)))
 			{
-				$args[$k] = $row[$v] ? $GLOBALS['TL_LANG']['MSC']['yes'] : $GLOBALS['TL_LANG']['MSC']['no'];
+				$args[$k] = ($row[$v] ?? null) ? $GLOBALS['TL_LANG']['MSC']['yes'] : $GLOBALS['TL_LANG']['MSC']['no'];
```

Comment:
This seems unrelated.

Reply:
it is the last remaining place in this method that does not check if `isset($row[$v])`. All the other places have been fixed before, so this should be fixed as well.